src/Goerli-DssSpell.t.base.sol

```diff
@@ -2071,7 +2100,8 @@ contract GoerliDssSpellTestBase is DSTest, DSMath {
 
         (,,,, uint256 dust) = vat.ilks(_ilk);
         dust /= RAY;
-        uint256 amount = 2 * dust * WAD / (_isOSM ? getOSMPrice(pip) : uint256(DSValueAbstract(pip).read()));
+        uint256 amount = 2 * dust * 10**token.decimals() / (_isOSM ? getOSMPrice(pip) : uint256(DSValueAbstract(pip).read()));
+        uint256 amount18 = token.decimals() == 18 ? amount : amount * 10**(18 - token.decimals());
```

Comment:
I think is better you leave the math as it was and rename that variable to `amount18` and then divide that value by 10 ^ (decimals difference). It should be better for rounding when dividing by the price.

Reply:
I'll check this one in detail according to what you share in chat, for now just ignore it.

Reply:
Ok yeah it's fine as it is because it will truncate the value by the token decimals precision when doing the join.